src/unwinder/i386/unwinder_i386.cpp

```diff
@@ -105,8 +105,19 @@ OOPStackUnwinderX86::VirtualUnwind(
     ENUM_CALLEE_SAVED_REGISTERS();
 #undef CALLEE_SAVED_REGISTER
 
-    ContextRecord->Esp = rd.SP - codeInfo.GetCodeManager()->GetStackParameterSize(&codeInfo);
-    ContextRecord->ResumeEsp = rd.SP;
+    SIZE_T paramSize = codeInfo.GetCodeManager()->GetStackParameterSize(&codeInfo);
+    SIZE_T paddingSize = 0;
+
+#ifdef UNIX_X86_ABI
+    // On UNIX_X86_ABI, function call may have stack alignment padding.
+    if (paramSize % 16 != 0)
```

Comment:
@janvorli As I understand, @seanshpark's patch adjusts ``STACK_ALIGN`` in JIT as 16 for x86/Linux. and this change will enforce each prolog to emit a 16-byte align frame. 

@seanshpark Could you let me know if there is something wrong?